src/Rendering/geometryBufferRenderer.ts

```diff
@@ -431,6 +431,10 @@ export class GeometryBufferRenderer {
             for (index = 0; index < alphaTestSubMeshes.length; index++) {
                 renderSubMesh(alphaTestSubMeshes.data[index]);
             }
+

```

Comment:
I think this is ok but should not be on by default
So I would suggest adding a property to enable it

Thoughts?